package.json

```diff
@@ -57,7 +57,7 @@
     "@hot-loader/react-dom": "^16.13.0",
     "app-module-path": "^2.2.0",
     "async": "^2.6.3",
-    "axios": "^0.15.3",
+    "axios": "^0.19.2",
```

Comment:
Have you double checked that this doesn't break anything? Just want to make sure :)

Reply:
I checked for breaking changes. I had run npm audit fix but I'll go back to the other package.json to be safe

Reply:
great, thanks!